keylime/tenant.py

```diff
@@ -642,66 +644,77 @@ def do_status(self, listing=False, returnresponse=False, bulk=False):
+            response['results'][agent_uuid]['operational_state'] = \
+                states.state_to_str(states.REGISTERED)
+            return response
+        elif response['code'] == 200:
+            if not listing:
+                if not bulk:
+                    operational_state = states.state_to_str(
+                        response['results'][agent_uuid]['operational_state'])
+                    response['results'][agent_uuid]['operational_state'] = operational_state
+
+                    logger.info("Agent Info:\n%s" % json.dumps(response["results"]))
 
-        if response.status_code != 200:
-            logger.error("Status command response: %s. Unexpected response from Cloud Verifier.", response.status_code)
-            sys.exit()
-        else:
-            response_json = response.json()
-            if not returnresponse:
-                if not listing:
-                    if not bulk:
-                        operational_state = response_json["results"]["operational_state"]
-                        logger.info('Agent Status: "%s"', states.state_to_str(operational_state))
-                        logger.info('Agent severity level: "%s"', response_json["results"]["severity_level"])
-                        logger.info('Agent last event id: "%s"', response_json["results"]["last_event_id"])
-                    else:
-                        for agent in response_json["results"].keys():
-                            response_json["results"][agent]["operational_state"] = states.state_to_str(response_json["results"][agent]["operational_state"])
-                        logger.info("Bulk Agent Info:\n%s" % json.dumps(response_json["results"]))
                 else:
-                    agent_array = response_json["results"]["uuids"]
-                    logger.info('Agents: "%s"', agent_array)
+                    for agent in response["results"].keys():
+                        response["results"][agent]["operational_state"] = \
+                            states.state_to_str(response["results"][agent][
+                                                    "operational_state"])
+                    logger.info("Bulk Agent Info:\n%s" % json.dumps(response["results"]))
             else:
-                return response_json["results"]
+                agent_array = response["results"]["uuids"]
+                logger.info('Agents: "%s"', agent_array)
 
-        return None
+            return response
+        else:
+            logger.error("Status command response: %s. Unexpected response from Cloud Verifier.", response['code'])
+            return response
 
     def do_cvdelete(self, verifier_check=True):
         """Delete agent from Verifier."""
         if verifier_check:
-            agent_json = self.do_status(listing=False, returnresponse=True)
+            agent_json = self.do_status()
+            if agent_json['code'] != 200:
+                logger.error("Could not get status of agent %s from "
```

Comment:
If the `"verifier."` string is hardcoded, why not put it into the original string? So it would have something like `verifier.%s`.